### ISSUE TITLE
Dani needs Delegating rights on the Giveth 2 Campaign

### ISSUE THREAD

**Issue by Danibelle** [Giveth/giveth-dapp#2072]
Can a Campaign have multiple Owners? I think so, Griff and I shared on some Decentral Campaigns..

I need to be able to delegate payments from the [Giveth 2 Campaign](https://beta.giveth.io/campaign/giveth-2) to Milestones created on it!

The "Delegate Funds" option comes up for me, but only for funds from the Gaia Gives DAC (which I created).

**Comment by aminlatifi** [comment]
@Danibelle No, only one user (owner) can delegate from Campaign. In case you want to become owner I can help you. 
Note co-owner role is able to only approve milestones.
